src/Compilers/VisualBasic/Portable/Parser/BlockContexts/CompilationUnitContext.vb

```diff
@@ -470,7 +471,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Syntax.InternalSyntax
                 End If
 
                 If reportAnError Then
-                    rewritten = _parser.ReportFeatureUnavailable(Feature.RegionsEverywhere, rewritten)
+                    rewritten = Feature.RegionsEverywhere.ReportFeatureUnavailable(_parser.Options, rewritten)
```

Comment:
Would it make sense to make `ReportFeatureUnavailable` an extension on syntax node instead? `rewritten.ReportFeatureUnavailable(Feature.RegionsEverywhere, _parser.Options)`